relay.go

```diff
@@ -307,7 +309,7 @@ func (r *Relayer) Receive(f *Frame, fType frameType) (sent bool, failureReason s
 	if item.tomb || (finished && !stopped) {
 		// Item has previously timed out, or is in the process of timing out.
 		// TODO: metrics for late-arriving frames.
-		return true, ""
+		return false, ""
```

Comment:
should a `failureReason` be returned here? the current caller of `Receive` uses the failure reason if `!sent`, so I think it's worth double checking how this path should work (what metrics should be emitted, etc)